### PR TITLE
refactor(fast-foundation): remove redundant role

### PR DESCRIPTION
**Pull request by yinonov**
this might seem pretentious to simply delete a line but, removing it makes no difference.
the `role` @attr decorated property member (by its default behavior) already reflects to the DOM host, which makes the removed line redundant in template.

https://github.com/microsoft/fast/blob/1d230aa72f4802bb309c4679bf5b2fa27d385e7b/packages/web-components/fast-foundation/src/menu-item/menu-item.ts#L115-L116

is there any other reason why this should stay?

### CODE REVIEW

**Comment by chrisdholt** [comment]
Thanks @yinonov - can you add change files for this PR?

**Comment by yinonov** [comment]
> Thanks @yinonov - can you add change files for this PR?

done. also added awaits to pass race conditions

**Review by EisenbergEffect — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

File: change/@microsoft-fast-foundation-656e7737-e8d8-4b21-a86d-47dbe895535b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "removed redundant role setting in fast foundation menu-item template",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "yinon@hotmail.com",
+  "dependentChangeType": "none"
```
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-656e7737-e8d8-4b21-a86d-47dbe895535b.json:6]:
```suggestion
  "dependentChangeType": "prerelease"
```

File: change/@microsoft-fast-foundation-656e7737-e8d8-4b21-a86d-47dbe895535b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-656e7737-e8d8-4b21-a86d-47dbe895535b.json:2]:
```suggestion
  "type": "prerelease",
```

**Review by nicholasrice — COMMENTED** [review]
Since there *are* changes to distributed code (even though it should have no effect) I think we should still increment the package version.

**Comment by yinonov** [comment]
> Since there _are_ changes to distributed code (even though it should have no effect) I think we should still increment the package version.

sure, your call. thank you

File: packages/web-components/fast-foundation/src/menu-item/menu-item.spec.ts [hunk 3]
```diff
@@ -36,6 +36,8 @@ describe("Menu item", () => {
 
         await connect();
 
+        await Updates.next();
+
```
Comment (yinonov) [inline packages/web-components/fast-foundation/src/menu-item/menu-item.spec.ts:40]:
```suggestion
```

File: packages/web-components/fast-foundation/src/menu-item/menu-item.spec.ts [hunk 4]
```diff
@@ -36,6 +36,7 @@ describe("Menu item", () => {
 
         await connect();
 
+        await Updates.next();
```
Comment (yinonov) [inline packages/web-components/fast-foundation/src/menu-item/menu-item.spec.ts:39]:
```suggestion
        await Updates.next();

```

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**